Replace broken image with Data URI

Paul-Hebert opened the pull request:
## Overview

The footer was hotlinking an image from the Cloud Four site. That image had been moved to a CDN and was no longer at the original location. It seemed like this image should live in this repo to avoid similar issues in the future, but I couldn't figure out the best place to put the image in the repo. The `images` directory seems like it's meant to house PWA icons.

I saw the cloud background was an SVG Data URI, so I took the same approach here.

## Screenshots

<img width="1440" alt="Screen Shot 2020-01-30 at 1 35 28 PM" src="https://user-images.githubusercontent.com/5798536/73450312-68f70b00-4365-11ea-981b-34e1a809873f.png">

## Testing

Review the site in a few different browsers, ensuring the image loads correctly

---

Fixes #188 

---

/CC @cloudfour/pwastats

tylersticka commented on _includes/contribute-story.html:
```diff
@@ -11,5 +11,9 @@
+    class="u-posAbsoluteBottomRight u-size2of5 u-sm-size1of3 u-sm-spaceRight1 u-md-size1of4"
+    src="data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' viewBox='0 0 140 70' width='140' height='70'%3E%3Cdefs%3E%3Cpolygon id='tree' points='0,70 30,0 60,70'/%3E%3Cuse id='tree-bg' xlink:href='%23tree' fill='%23249579'/%3E%3Cuse id='tree-fg' xlink:href='%23tree' fill='%2335C98D'/%3E%3Cg id='limbs' stroke='%23DBE5EA'%3E%3Cline ...
+    alt="Portland, Oregon landscape"
+  >
```
@Paul-Hebert Did you try replacing the `<img>` with an `<svg>` instead of making the SVG a data URI?

Paul-Hebert replied:
No, but I'd be happy to do that instead if you prefer. I'll do that now.

Paul-Hebert commented:
@tylersticka , I changed this to an inline SVG and added some inline comments explaining where this graphic is from, and why it is included the way it is. Mind taking another look?

tylersticka commented on _includes/contribute-story.html:
```diff
@@ -11,5 +11,52 @@
+    reasonable size. It does _not_ have a height attribute. Since our CSS isn't
+    modifying the height, a height attribute would remain in effect, shrinking
+    the SVG on large screens.
+  -->
```
This seems like a pretty verbose comment to output to the site? 🤔

If the inline SVG requires this much more explanation, I'm not sure it's preferable to the data URI?

Paul-Hebert replied:
That's fair. I guess I'm spoiled by templating engines and build tools that strip out comments like this 😅 

I'm not sure the comment is necessary. I'll take a look and see if there are parts that should be kept and shortened, or if the whole thing should be yanked out.

I don't have a strong preference between the inline SVG and the data URI.

Paul-Hebert replied:
Good call @tylersticka . Re-reading it, most of that comment was unnecessary. I stripped out most of it but left a couple lines explaining what the graphic was so devs don't need to parse the SVG to understand it. I'm happy to make further changes or remove if completely if you'd prefer. 

Mind taking another look? Thanks 🙂

tylersticka replied:
@Paul-Hebert I'm not super familiar with Jekyll, but I think it uses Liquid? Did you try wrapping the comment in [Liquid comment tags](https://shopify.github.io/liquid/tags/comment/)?

Paul-Hebert replied:
Oh good catch. Thanks. I was thrown off by the `.html` file ending. Lemme see...

Paul-Hebert replied:
Just switched it to a liquid comment 🙂
